frontend/best_practises.md

```diff
@@ -1,50 +1,51 @@
 # Flotilla frontend development best practises
 
-- [Flotilla frontend development best practises](#flotilla-frontend-development-best-practises)
```

Comment:
A lot of these changes to READMEs and whatnot are very nice, but would it not be good to seperate them into different commits?

Reply:
Only a suggestion, not a hard requirement. Just to generally split up the big commit somewhat